EasyPost/Services/ScanFormService.cs

```diff
@@ -25,7 +24,7 @@ internal ScanFormService(EasyPostClient client) : base(client)
         public async Task<ScanForm> Create(List<Shipment> shipments)
         {
             Dictionary<string, object> parameters = new Dictionary<string, object> { { "shipments", shipments } };
-            parameters = parameters.Wrap("scan_form");
+            // parameters = parameters.Wrap("scan_form"); // TODO: Remove if not necessary
```

Comment:
This should not be needed (probably one of the only ones) and the comment can be removed

Reply:
Resolved